pkg/antctl/antctl.go

```diff
@@ -81,6 +82,29 @@ var CommandList = &commandList{
 			},
 			transformedResponse: reflect.TypeOf(version.Response{}),
 		},
+		{
+			use:   "podmulticaststats",
+			short: "Show multicast statistics",
+			long:  "Show multicast statistics of ${component}",
+			example: `  Show the multicast statistics per pod inside node
```

Comment:
pod -> Pod

Reply:
inside node -> on this Node

Reply:
Probably "Show multicast traffic statistics of all local Pods on the Node"

We should add an example for "Showing multicast traffic statistics of a given Pod".

Reply:
updated